invokeai/app/api/routers/model_manager.py

```diff
@@ -816,3 +814,61 @@ async def get_starter_models() -> list[StarterModel]:
         model.dependencies = missing_deps
 
     return starter_models
+
+
+@model_manager_router.get(
+    "/model_cache",
+    operation_id="get_cache_size",
+    response_model=float,
+    summary="Get maximum size of model manager RAM or VRAM cache.",
+)
+async def get_cache_size(cache_type: CacheType = Query(description="The cache type", default=CacheType.RAM)) -> float:
+    """Return the current RAM or VRAM cache size setting (in GB)."""
+    cache = ApiDependencies.invoker.services.model_manager.load.ram_cache
+    return cache.max_cache_size if cache_type == CacheType.RAM else cache.max_vram_cache_size
```

Comment:
To make this robust to future changes, this would be better:
```suggestion
    if cache_type == CacheType.RAM:
        return cache.max_cache_size
    elif cache_type == CacheType.VRAM:
        return cache.max_vram_cache_size
    else:
        raise ValueError(f"Unexpected {cache_type=}.")
```